client/src/components/common/Card.js

```diff
@@ -76,7 +77,7 @@ const PatientCard = (props) => {
           Reset Layout
         </MenuItem>
       </Menu>
-      <Card className={classes.root} variant="outlined">
+      <Card className={isDraggable ? classes.root : classes.minHeightCard} variant="outlined">
```

Comment:
I am not sure if ternary is a good choice here. You could use something like this:

```
 className={clsx({
            [classes.modalConentBelow]: true, //always apply
            [classes.contentWithLoading]: isLoading //only when isLoading === true
          })}
```

Reply:
In my opinion, third party libraries should be avoided especially when you can apply simple conditional checks using javascript.

Reply:
Agree but it's a tiny dependency and also suggested by material ui. 
https://material-ui.com/getting-started/faq/#whats-the-clsx-dependency-for

Also it's already included on our projct.

Reply:
Fixed it.